### PR TITLE
OL7 DISA STIG v2r1 update

### PR DESCRIPTION
**Pull request by freddieRv**
#### Description:

- Updated OL7 STIG profile to conform with OL7 DISA STIG v2r1. Updated CCIs and SRGs where necessary. Generated new stig overlay and added DISA STIG OL7 v2r1 reference manual. 

#### Rationale:

- STIG profile for OL7 had not been updated since it was added.

### CODE REVIEW

File: linux_os/guide/system/permissions/partitions/mount_option_dev_shm_nodev/rule.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ identifiers:
 references:
     cis@rhel8: 1.1.5
     cis@ubuntu1804: 1.1.14
-    stigid@ol7: OL07-00-021022
+    stigid@ol7: OL07-00-021024
```
Comment (ggbecker) [inline linux_os/guide/system/permissions/partitions/mount_option_dev_shm_nodev/rule.yml:27]: Duplicated STIGIDs pose a bit of a problem. when results are translated into STIG viewer format, for example, only one of the rules will have the results incorporated. Also, `stig_overlay.xml` will have one of these three rules. I know that DISA has deliberately merged these rules into one single requirement, but that's the opposite we ultimately want to achieve. We want to have a single configuration item per rule. I have tried to merge these three rules together in the past but with no success, see the discussion here: https://github.com/ComplianceAsCode/content/pull/6087

I'm not complaining about this change itself, but more about explaining the context behind the problems that people might have with STIG results and mapping.

**Comment by ggbecker** [comment]
I have a small remark regarding STIG ID, otherwise, it looks good.

File: linux_os/guide/system/permissions/partitions/mount_option_dev_shm_nodev/rule.yml (hunk 1 above)
Comment (freddieRv) [reply linux_os/guide/system/permissions/partitions/mount_option_dev_shm_nodev/rule.yml:27]: Got it. I'll follow the same approach and leave the old ids.

**Comment by ggbecker** [comment]
Changes look good to me. That's a good improvement for OL users. With the new STIG benchmark v2r1, the content now can generate compatible STIG viewer results to be used with the latest version since DISA has come up with new identifiers for the requirements.

**Review by ggbecker — APPROVED** [review]

**Pull request merged**